weblate/trans/models/unit.py

```diff
@@ -989,6 +1052,7 @@ def update_from_unit(  # noqa: C901,PLR0914
                 only_save=True,
                 update_fields=["location", "explanation", "note", "position"],
             )
+            # TODO: delete pending changes and details for the unit in that case ?
```

Comment:
Should existing pending changes be discarded in this case? IIUC, this means unflushed changes to `explanation` will be lost.